enterprise_access/apps/subsidy_request/psg_tasks.py

```diff
@@ -0,0 +1,108 @@
+"""
+Tasks for subsidy requests app.
+"""
+
+import logging
+from datetime import datetime
+
+from celery import shared_task
+from django.apps import apps
+from django.conf import settings
+from requests.exceptions import HTTPError
+
+from enterprise_access.apps.api_client.braze_client import BrazeApiClient
+from enterprise_access.apps.api_client.discovery_client import DiscoveryApiClient
+from enterprise_access.apps.api_client.lms_client import LmsApiClient
+from enterprise_access.apps.subsidy_request.constants import SubsidyRequestStates
+from enterprise_access.tasks import LoggedTaskWithRetry
+from enterprise_access.utils import get_subsidy_model
+
+logger = logging.getLogger(__name__)
+
+@shared_task(base=LoggedTaskWithRetry)
+def psg_send_admins_email_with_new_requests_task(enterprise_customer_uuid):
```

Comment:
Can we make this task work for a set of `SubsidyRequestCustomerConfiguration` as opposed to just one? Running this on each `SubsidyRequestCustomerConfiguration` could be wasteful, and a bulk task would be more efficient.